src/implementations/twig/components/video/video.html.twig

```diff
@@ -38,6 +38,7 @@
 {% set _title = title|default('') %}
 {% set _sr_video_player = sr_video_player|default('') %}
 {% set _sr_video_label = sr_video_label|default('') %}
+{% set _extra_attributes = '' %}
```

Comment:
shouldn't it be an empty array instead of a string?

Reply:
no, this is the local variable, it's a string